frontend/packages/dev-console/src/components/topology2/components/nodes/BaseNode.tsx

```diff
@@ -62,6 +64,14 @@ const BaseNode: React.FC<BaseNodeProps> = ({
   useAnchor(EllipseAnchor);
   const cx = element.getBounds().width / 2;
   const cy = element.getBounds().height / 2;
+  const resourceModel = modelFor(referenceFor(element.getData().data.donutStatus.dc));
+  const editAccess = useAccessReview({
+    group: resourceModel.apiGroup,
+    verb: 'patch',
+    resource: resourceModel.plural,
+    name: element.getData().data.donutStatus.dc.metadata.name,
+    namespace: element.getData().data.donutStatus.dc.metadata.namespace,
```

Comment:
Using `donutStatus` as the main resource is very odd.
I guess we don't have any other data point to read from here :(

Reply:
We have the only ```donutStatus``` where resource data exist.

Reply:
`element.getData().resources.obj.metadata` is available

Reply:
Done.